pybind/linalg_py.cpp

```diff
@@ -270,6 +270,79 @@ void linalg_binding(py::module &m) {
       return cytnx::linalg::ExpH(Tin, a, b);
     },
     py::arg("Tin"), py::arg("a"), py::arg("b") = cytnx_int16(0));
+  // m_linalg.def("ExpH", [](const UniTensor &Tin, const cytnx::Scalar &a, const cytnx::Scalar &b) {
+  // return cytnx::linalg::ExpH(Tin, a, b); }, py::arg("Tin"), py::arg("a"), py::arg("b") = 0);
+  m_linalg.def(
+    "ExpH",
+    [](const UniTensor &Tin, const py::numpy_scalar<std::complex<double>> &a,
+       const py::numpy_scalar<std::complex<double>> &b) {
+      return cytnx::linalg::ExpH(Tin, (cytnx::cytnx_complex128)a);
```

Comment:
Is c-style cast a proper way to cast numpy_scalar? Or it should be `a.cast<T>()`? If c-style cast works , I think static_cast is a better option for C++.